react/.eslintrc.js

```diff
@@ -21,7 +21,9 @@ module.exports = {
                 'no-shadow': 'off',
                 '@typescript-eslint/no-shadow': [ 'error' ],
                 'typescript-sort-keys/interface': 'error',
-                'typescript-sort-keys/string-enum': 'error'
+                'typescript-sort-keys/string-enum': 'error',
+                'object-curly-spacing': 'off',
```

Comment:
On the contrary, we want this ON and to give an error, including on TS.

Reply:
Is this a requirement when we are mixing it with TS code?

Reply:
Yes. We need to disable the default rule and enable the @typescript-eslint rule

Reply:
👍